atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -182,6 +184,10 @@ default String getKeyspaceOrThrow() {
      */
     Optional<SslConfiguration> sslConfiguration();
 
+    default CqlClientFactory cqlClientFactory() {
```

Comment:
you want to mark this as `@Value.Default` otherwise consumers will have to override it

Reply:
Added.